pages/index.tsx

```diff
@@ -207,6 +231,54 @@ serve((req) => new Response("Hello World\\n"), { port: 8000 });`;
             <a class="link" href="/x">deno.land/x</a>.
           </p>
         </div>
+        <div class="max-w-screen-sm mx-auto px-4 sm:px-6 md:px-8 mt-20">
+          <a class="hover:underline" href="#toolchain">
+            <h3 class="font-bold text-xl" id="toolchain">
+              Built-in Toolchain
+            </h3>
+          </a>
+          <p class="my-4 text-gray-700">
+            Deno comes with a robust{" "}
+            <a class="link" href="/manual/tools">set of tools</a>, so you can
+            spend less time searching and evaluating third party modules, and
+            more time writing code and being productive. Here are a few
+            examples.
+          </p>
+          <p class="my-4 text-gray-700">
+            <a class="link" href="/manual/tools/linter">Lint</a>{" "}
+            all JS/TS files in the current directory and subdirectories:
+          </p>
+          <p>
+            <CodeBlock
+              code={"deno lint\nChecked `n` files"}
+              language="bash"
+            />
+          </p>
+          <p class="my-4 text-gray-700">
+            <a class="link" href="/manual/tools/formatter">Format</a>{" "}
+            all supported files in the current directory and subdirectories:
+          </p>
+          <p>
+            <CodeBlock
+              code={"deno fmt\nChecked `n` files"}
+              language="bash"
+            />
+          </p>
+          <p class="my-4 text-gray-700">
+            <a class="link" href="/manual/tools/bundler">Bundle</a>{" "}
+            your project into a single JS file, including all dependencies:
+          </p>
+          <p>
+            <CodeBlock
+              code={denoBundleExample}
+              language="bash"
+            />
+          </p>
```

Comment:
I'd say bundler is quite controversial, since there's a mismatch in user expectation of what it can do. I suggest to replace it with `deno test` example